src/makeRobot.js

```diff
@@ -38,7 +44,60 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name: name,
+    wheels: wheels,
+    version: version,
+    coords: {
+      x: 0, y: 0,
+    },
+
+    get info() {
+      return 'name: ' + name + ', chip version: ' + version
+        + ', wheels: ' + wheels;
+    },
```

Comment:
```suggestion
      return `name: ${this.name}, `
      + `chip version: ${this.version}, `
      + `wheels: ${this.wheels}`;
    },
```
You could use interpolation here